engine/core/IgeScene2d.ts

```diff
@@ -8,9 +8,7 @@ import { registerClass } from "@/engine/igeClassStore";
  */
 export class IgeScene2d extends IgeEntity {
 	classId = "IgeScene2d";
-	_shouldRender: boolean;
 	_autoSize: boolean;
-	_pointerAlwaysInside: boolean;
```

Comment:
This looks incorrectly removed?

Reply:
This is defined on IgeObject (should it be? I think a review of IgeObject properties could be worthwhile to reduce memory consumption in large object graphs - I have some thoughts on this)

Reply:
Cool. Yeah it might make sense to move it to IgeEntity but that won't make much difference to the point you made about memory consumption. Let's keep this change as-is and we can re-visit where it's declared later.